client/src/App.tsx

```diff
@@ -21,6 +23,11 @@ function App() {
     currency: '',
   });
   const [fxRates, setFxRates] = useState<FxRates>({CAD: 1, EUR: 1, USD: 1});
+  const [isSnackbarOpen, setIsSnackbarOpen] = useState(false);
+  const [snackbarMessage, setSnackbarMessage] = useState('');
+  const [alertColor, setAlertColor] = useState<AlertColor | undefined>(
+    undefined
```

Comment:
I think you can remove this undefined cos having nothing in the brackets is probably treated the same as explicit undefined